test/pkg/pkg.config.json

```diff
@@ -7,10 +7,10 @@
             "../../node_modules/pino-abstract-transport/index.js"
         ],
         "targets": [
-            "node14",

```

Comment:
Remove node14 because it doesn't support `node:assert/strict`.
I don't know if we want to keep node16 and node18 or remove them.

Reply:
Remove them and add 24.

Reply:
Removed them, but node24 isn't supported yet.
<img width="826" alt="Screenshot 2025-05-09 at 14 09 35" src="https://github.com/user-attachments/assets/314c68bb-b60d-4801-9da2-b460b4faebe3" />